ChemistryLib/PhreeqcIOData/CreateKnobs.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateKnobs.h"
+#include "BaseLib/ConfigTree.h"
+#include "Knobs.h"
+
+namespace ChemistryLib
+{
+namespace PhreeqcIOData
+{
+Knobs createKnobs(BaseLib::ConfigTree const& config)
+{
+    auto const max_iterations =
+        //! \ogs_file_param{prj__chemical_system__knobs__max_iterations}
+        config.getConfigParameter<int>("max_iter", 100);
```

Comment:
Didn't you change that part to be mandatory? At least I've seen the project files being updated...

Reply:
Dropped the default values...